modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -145,7 +145,22 @@ Configuration example:
 ----
 | No
 | `target` | The CloudHub 2.0 target name to deploy the app to. +
-Specify either a shared space or a private space available in your Deployment Target values in CloudHub 2.0. See xref:cloudhub-2::ch2-features.adoc[Features of CloudHub 2.0] for a detailed description on shared and private spaces. | Yes
+Specify either a shared space or a private space available in your Deployment Target values in CloudHub 2.0. Below are target values for shared space:
+
+* `Cloudhub-US-East-1` (US East, N. Virginia)
+* `Cloudhub-US-East-2` (default; US East, Ohio)
```

Comment:
I wouldn't mention this as default as it may be changed by the customer

Reply:
@juanmanuelmenendez - This is default in case you do not permission to select region while creating account/org on anypoint platform. It works same way as cloudhub 1.0  (This is default for trial accounts)